motoman_mhj_support/urdf/mhj_macro.xacro

```diff
@@ -0,0 +1,165 @@
+    </link>
+    <link name="${prefix}link_3_u">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_mhj_support/meshes/mhj/visual/link_3_u.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_mhj_support/meshes/mhj/collision/link_3_u.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_4_r">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_mhj_support/meshes/mhj/visual/link_4_r.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_mhj_support/meshes/mhj/collision/link_4_r.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_5_b">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_mhj_support/meshes/mhj/visual/link_5_b.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_mhj_support/meshes/mhj/collision/link_5_b.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_6_t">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_mhj_support/meshes/mhj/visual/link_6_t.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_mhj_support/meshes/mhj/collision/link_6_t.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}tool0"/>
+    <!-- end of link list -->
+
+    <!-- joint list -->
+    <joint name="${prefix}joint_1_s" type="revolute">
+        <parent link="${prefix}base_link"/>
+        <child link="${prefix}link_1_s"/>
+        <origin xyz="0 0 0.240" rpy="0 0 0" />
+        <axis xyz="0 0 1" />
+        <limit lower="${radians(-160)}" upper="${radians(160)}" effort="44.72" velocity="${radians(160)}"/>
+    </joint>
+    <joint name="${prefix}joint_2_l" type="revolute">
+        <parent link="${prefix}link_1_s"/>
+        <child link="${prefix}link_2_l"/>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <axis xyz="0 1 0" />
+        <limit lower="${radians(-90)}" upper="${radians(110)}" effort="65.9" velocity="${radians(130)}"/>
+    </joint>
+    <joint name="${prefix}joint_3_u" type="revolute">
+        <parent link="${prefix}link_2_l"/>
+        <child link="${prefix}link_3_u"/>
+        <origin xyz="0 0 0.275" rpy="0 0 0" />
+        <axis xyz="0 -1 0" />
+        <limit lower="${radians(-45)}" upper="${radians(210)}" effort="65.9" velocity="${radians(200)}"/>
```

Comment:
According to [this specsheet](https://yaskawaindia.in/wp-content/uploads/2012/10/MOTOMAN-MHJ.pdf), the limits for `U` are: `[-290; 105]`.

Is the specsheet incorrect?

Reply:
There are a few spec sheet where they give the range of motion of the U-axis relative to the ground and it is the compound angle of the L and U range.  So it is not incorrect and looks good from a marketing perspective but not useful for us.

Reply:
You are right about the color, that was an over sight.  I'll get that fixed.